doc/code/qml_qaoa.rst

```diff
@@ -1,6 +1,50 @@
 qml.qaoa
 ========
 
+This module provides a collection of methods that help in the construction of
+QAOA workflows.
+
+Here is an example of the PennyLane QAOA functionality being used to solve the
+`MaxCut <https://en.wikipedia.org/wiki/Maximum_cut>`__ problem:
+
+.. code-block:: python3
+
+    import pennylane as qml
+    from pennylane import qaoa
+    from networkx import Graph
+
+    # Defines the wires and the graph on which MaxCut is being performed
+
```

Comment:
```suggestion
```
Here and throughout, I think it's best no to leave a line between a comment and its corresponding code